ow_lander/scripts/dig_circular_action_client.py

```diff
@@ -4,45 +4,71 @@
 # Research and Simulation can be found in README.md in the root directory of
 # this repository.
 
+# Default trenching values
+# goal.x_start = 1.65
+# goal.y_start = 0
+# goal.depth = 0.01
+# goal.parallel = False
+# goal.ground_position = constants.DEFAULT_GROUND_HEIGHT
```

Comment:
A slightly cleaner way to code defaults is to make these variables, and then use the variables in the add_argument calls below.  This way the default constants are coded in only one place.